samples/Exporters/Console/TestRedis.cs

```diff
@@ -28,8 +27,17 @@ internal class TestRedis
     {
         internal static object Run(string zipkinUri)
         {
-            // connect to the server
-            var connection = ConnectionMultiplexer.Connect("localhost:6379");
+            /*
+             * Setup redis service inside local docker.
+             * docker run --name opentelemetry-redis-test -d -p 6379:6379 redis
+             * docker exec -it opentelemetry-redis-test sh
```

Comment:
can you put a comment about this - only the 1st line is required. If there are issues connecting to docker for the host, then do the remaining part.
It look like after a restart, I am able to connect without the bind to 0.0.0.0

Reply:
ref: https://stackoverflow.com/questions/41371402/connecting-to-redis-running-in-docker-container-from-host-machine

Reply:
i will push soon the following block:
```
/*
             * Setup redis service inside local docker.
             * docker run --name opentelemetry-redis-test -d -p 6379:6379 redis
             * 
             * If you face any issue with the first command, do the following ones:
             * docker exec -it opentelemetry-redis-test sh
             * redis-cli
             * set bind 0.0.0.0
             * save
             */
```